cypress-tests/cypress/e2e/happyPath/platform/ceTestcases/apps/appImportAndExport.cy.js

```diff
@@ -229,17 +264,21 @@ describe("App Import Functionality", () => {
           });
```

Comment:
- Add validations for the version input checkbox and  verify the app creation time
- Add validations for Tooljet schema checkbox and text aswell